timescaledb/how-to-guides/continuous-aggregates/create-index.md

```diff
@@ -31,27 +35,21 @@ CREATE MATERIALIZED VIEW conditions_daily
 ```
 
 ## Manually create and drop indexes
-You can manually create and drop indexes. To do so, you need to know the name of
-your materialized hypertable. To find the name, see the instructions in the
-[managing materialized hypertables][materialized-hypertable-name] section.
-
-<highlight type="note">
-The name you give when you run `CREATE MATERIALIZED VIEW` is the view name. The
-continuous aggregate's data is stored in a materialized hypertable, which is
-automatically created and named.
-</highlight>
-
-You can then use a regular PostgreSQL statement to create or drop an index on
-the hypertable. For example, to create an index on `avg_temp` for a materialized
-hypertable named `_timescaledb_internal._materialized_hypertable_2`:
+
+You can use a regular PostgreSQL statement to create or drop an index on a
+continuous aggregate. For example, to create an index on `avg_temp` for a
+materialized hypertable named `weather_daily`:
+
```

Comment:
Would be nice if we have a note here about `DROP INDEX` because to work the user should use the internal schema `_timescaledb_internal.index_name` because this schema is where the materialization hypertable is stored. So, for example:

```sql
CREATE TABLE conditions (
       time TIMESTAMPTZ NOT NULL,
       location TEXT NOT NULL,
       temperature DOUBLE PRECISION NULL
);

SELECT create_hypertable('conditions', 'time');

CREATE MATERIALIZED VIEW conditions_daily
WITH (timescaledb.continuous, timescaledb.materialized_only = false) AS
SELECT location,
       time_bucket(INTERVAL '1 day', time) AS bucket,
       AVG(temperature)
  FROM conditions
GROUP BY location, bucket
WITH NO DATA;

CREATE INDEX index_avg ON conditions_daily (avg);

DROP INDEX _timescaledb_internal.index_avg;
```